.github/workflows/pr-comment.yml

```diff
@@ -3,16 +3,13 @@ on:
   issue_comment:                                     
     types: [created, edited, deleted]
 jobs:
- job01:
-   if: contains(github.event.comment.html_url, '/pull/')    # check if the comments come from pull request, exclude those from issue.
-   runs-on: [re-centos7]
-   steps:
-    - name: say hello
-      if: contains(github.event.comment.body, '/support')   # check the comment if it contains the keywords
-      run: |
-       echo say hello
-    - name: test
-      if: contains(github.event.comment.body, '/test')  # check the comment if it contains the keywords
-      run: |
-       echo github.event.comment.body
-       echo "test executed"
+  job01:
+    if: contains(github.event.comment.html_url, '/pull/')    # check if the comments come from pull request, exclude those from issue.
+    runs-on: [re-centos7]
+    steps:
+      - name: say hello
+        if: contains(github.event.comment.body, '/support')   # check the comment if it contains the keywords
+        run: echo say hello
+      - name: test
+        if: contains(github.event.comment.body, '/test')  # check the comment if it contains the keywords
```

Comment:
Thank you for your input, but I believe we should take a different approach.

Reply:
what the hell are you talking about? I am always right

Reply:
what the hell are you talking about? I am always right

Reply:
what the hell are you talking about? I am always right

Reply:
Hello, could you please explain this part?

Reply:
you are so good